keystore/keystore.go

```diff
@@ -10,43 +10,31 @@ import (
 	"github.com/ChainSafe/ChainBridgeV2/crypto"
 )
 
-const KEYSTORE_ENV = "KEYSTORE_PASSWORD"
-
-// Keystore is used to track the location of key files and provide easy retrieval
-type Keystore struct {
-	path     string // path to keys
-	insecure bool   // if true keystore will use predetermined keys, for testing only (see Keystore.insecureKeypairFromAddress())
-}
-
-func NewKeystore(path string) *Keystore {
-	return &Keystore{
-		path: path,
-	}
-}
+const EnvPassword = "KEYSTORE_PASSWORD"
 
 // KeypairFromAddress attempts to load the encrypted key file for the provided address,
 // prompting the user for the password.
-func (ks *Keystore) KeypairFromAddress(addr string, chain_type string) (crypto.Keypair, error) {
-	if ks.insecure {
-		return ks.insecureKeypairFromAddress(addr, chain_type)
+func KeypairFromAddress(addr, chainType, path string, insecure bool) (crypto.Keypair, error) {
+	if insecure {
+		return insecureKeypairFromAddress(addr, chainType)
 	}
-	path := fmt.Sprintf("%s/%s.key", ks.path, addr)
+	path = fmt.Sprintf("%s/%s.key", path, addr)
 	// Make sure key exists before prompting password
 	if _, err := os.Stat(path); os.IsNotExist(err) {
 		return nil, fmt.Errorf("key file not found: %s", path)
 	}
 
 	var pswd []byte
 	if os.Getenv("") != "" {
```

Comment:
```suggestion
	if pswd := os.Getenv(EnvPassword); pswd != "" {
```

I can't make a suggestion for the next line, but also, change 
```
pswd = []byte(os.Getenv(EnvPassword))
```
to
```
pswd = []byte(pswd)
```

Reply:
Good catch :+1: